Terminal.Gui/Windows/Dialog.cs

```diff
@@ -116,17 +116,104 @@ internal int GetButtonsWidth ()
 			}
 			return buttons.Select (b => b.Bounds.Width).Sum () + buttons.Count - 1;
 		}
+		/// <summary>
+		/// Determines the horizontal alignment of the Dialog buttons.
+		/// </summary>
+		public enum ButtonAlignments {
+			/// <summary>
+			/// Center-aligns the buttons (the default).
+			/// </summary>
+			Center = 0,
+
+			/// <summary>
+			/// Justifies the buttons
+			/// </summary>
+			Justify,
+
+			/// <summary>
+			/// Left-aligns the buttons
+			/// </summary>
+			Left,
+
+			/// <summary>
+			/// Right-aligns the buttons
+			/// </summary>
+			Right
+		}
+		/// <summary>
+		/// Determines how the <see cref="Dialog"/> <see cref="Button"/>s are aligned along the 
+		/// bottom of the dialog. 
+		/// </summary>
+		public ButtonAlignments ButtonAlignment = Dialog.ButtonAlignments.Center;
```

Comment:
Should this be a property instead of a field so it is consistent with the other public properties.  And so we can add setter logic later if we wanted to without breaking changes.

![Alignment](https://user-images.githubusercontent.com/31306100/173179810-2fa81cbf-7d7a-4f73-84ab-170036d43e16.jpg)

Reply:
Entering 0 and then pressing "Add a button" also crashes.